### ISSUE TITLE
RFE: new release? 🤔 

### ISSUE THREAD

**Issue by kloczek** [rockowitz/ddcutil#464]
https://github.com/rockowitz/ddcutil/compare/v2.1.4...2.1.5-dev shows +700 commits since last release.
Do you have any plans to release new version soon? 🤔

**Comment by rockowitz** [comment]
Yes, release 2.1.5 is long overdue.  (Also, given the extent of changes it will become 2.2.0). The problem is that watching for display connection/disconnection in libddcutil has been "almost" done for some time now.  In retrospect, 2.1.5 should have been released without those changes, but that is now impractical.  I'm now awaiting feedback from key API users.
